src/main/java/org/folio/consortia/service/impl/TenantServiceImpl.java

```diff
@@ -149,4 +170,58 @@ private ConsortiaConfiguration createConsortiaConfigurationBody(String tenantId)
     configuration.setCentralTenantId(tenantId);
     return configuration;
   }
+
+  private void createShadowAdminUserWithPermissions(UUID userId) {
+    User userOptional = userTenantService.getUser(userId);
+    if (Objects.isNull(userOptional.getId())) {
+      userOptional = createUser(userId);
+    }
+    Optional<PermissionUser> permissionUserOptional = permissionsClient.get("userId==" + userOptional.getId())
+      .getPermissionUsers()
+      .stream()
+      .findFirst();
+    if (permissionUserOptional.isEmpty()) {
```

Comment:
if permissions exist, you should update with perms from file

Reply:
fixed

Reply:
now you are deleting all existed permissions and replaced with new permission set from file. I am not sure it is correct, @NikitaSedyx what do you think?

Reply:
@SerhiiNosko :- i am not deleting all exising permissions from user, That operation is just for not adding duplicate permission.